pkg/controller/jobs/pod/pod_webhook.go

```diff
@@ -53,7 +52,8 @@ const (
 	GroupTotalCountAnnotation    = "kueue.x-k8s.io/pod-group-total-count"
 	GroupFastAdmissionAnnotation = "kueue.x-k8s.io/pod-group-fast-admission"
 	GroupServingAnnotation       = "kueue.x-k8s.io/pod-group-serving"
-	SuspendedByParentAnnotation  = "kueue.x-k8s.io/pod-suspending-parent"
+	ParentFrameworkAnnotation    = "kueue.x-k8s.io/parent-framework"
```

Comment:
```suggestion
	PodParentFrameworkAnnotation    = "kueue.x-k8s.io/pod-parent-framework"
```
Since this annotation is pod-integration specifc.